nats-server/configuration/monitoring.md

```diff
@@ -520,6 +521,111 @@ The `/subsz` endpoint reports detailed information about the current subscriptio
 }
 ```
 
+### Account Information
+
+The `/accountz` endpoint reports information on a servers active accounts. 
+The default behavior is to return a list of all accounts known to the server. 
+
+**Endpoint:** `http://server:port/accountz`
+
+| Result | Return Code |
+| :--- | :--- |
+| Success | 200 \(OK\) |
+| Error | 400 \(Bad Request\) |
+
+#### Arguments
+
+| Argument | Values | Description |
+| :--- | :--- | :--- |
+| acc | account name | Include detail information for the requested account. Default is empty. When not set, a list of all accounts is included. |
```

Comment:
Might suggest `Include metrics for the specified account`

Reply:
incorporated, thanks